internal/rest/resources/cluster.go

```diff
@@ -51,6 +51,18 @@ func clusterPost(s *state.State, r *http.Request) response.Response {
 	// If we received a forwarded request, assume the new member was successfully added on the leader,
 	// and execute the new member hook.
 	if client.IsForwardedRequest(r) {
+		ctx, cancel := context.WithTimeout(s.Context, 30*time.Second)
```

Comment:
Why was 30s picked here?

Reply:
It's the same as the query timeout in the corresponding client function.

Reply:
Is that sufficient for slower machines?

Reply:
I haven't personally been able to test mich on slower machines, but I wouldn't think this would be a problem. Dqlite itself starts up quickly, it's the ceph and ovn setup that takes a while in MicroCloud at least. 

I stumbled upon this in MicroOVN where it causes a panic in the `OnNewMember` hook trying to use the database. But given that I haven't seen this error reported anywhere and clusters haven't been failing to form at this point, I'd say it's probably sufficient.

Reply:
OK thanks for confirming